### DISCUSSION TITLE
Cannot pull when using Rancher Desktop and wsl-vpnkit

### DISCUSSION THREAD

**Discussion by abcarcol** [rancher-sandbox/rancher-desktop#1962] [Q&A]
I am using wsl-vpnkit to provide network connectivity to the WSL 2 VM while connected to VPNs on the Windows host. However, when I try to pull any image with Rancher Desktop I get the following:

> nerdctl pull nginx
INFO[0000] trying next host                              error="failed to do request: Head \"https://registry-1.docker.io/v2/library/nginx/manifests/latest\": dial tcp: lookup registry-1.docker.io on 172.29.215.96:53: read udp 172.29.215.96:34076->172.29.215.96:53: read: connection refused" host=registry-1.docker.io
FATA[0000] failed to resolve reference "docker.io/library/nginx:latest": failed to do request: Head "https://registry-1.docker.io/v2/library/nginx/manifests/latest": dial tcp: lookup registry-1.docker.io on 172.29.215.96:53: read udp 172.29.215.96:34076->172.29.215.96:53: read: connection refused

> nslookup registry-1.docker.io
Server:  anycastdns.corp.xxx.com
Address:  10.5.5.5

Non-authoritative answer:
Name:    registry-1.docker.io
Addresses:  44.196.236.180
          54.85.133.123
          52.21.28.242
          52.72.252.48
          52.205.127.201
          54.86.228.181
          54.197.112.205
          34.203.135.183

**Comment by jcageman** [comment]
This is the same issue as in https://github.com/rancher-sandbox/rancher-desktop/discussions/1566 i believe. You try to workaround it with wsl-vpnkit, but i think this should be solved by rancher-desktop itself anyways. Also have this issue by the way.
